validate/BUILD

```diff
@@ -35,6 +35,7 @@ go_proto_library(
     name = "validate_go_proto",
     importpath = "github.com/envoyproxy/protoc-gen-validate/validate",
     proto = ":validate_proto",
+    gc_goopts = ["-trimpath=$(BINDIR)=>."],
```

Comment:
This is an important change for go plugins to work. see more info about this on: https://github.com/bazelbuild/rules_go/pull/2994

Reply:
the rules_go fix is now merged!